prow/scripts/kyma-testing.sh

```diff
@@ -119,7 +87,7 @@ function main() {
   export INSTALL_DIR=${TMP_DIR}
```

Comment:
you can remove this line. This was used by the Kyma-cli script for installation the binary to temporary directory.